resource-managers/kubernetes/core/src/test/scala/org/apache/spark/deploy/kubernetes/submit/ClientV2Suite.scala

```diff
@@ -169,30 +189,85 @@ class ClientV2Suite extends SparkFunSuite with BeforeAndAfter {
             .endMetadata()
         }
       })
-    when(initContainerComponentsProvider.provideContainerLocalizedFilesResolver())
+    when(initContainerComponentsProvider.provideContainerLocalizedFilesResolver(any[String]))
```

Comment:
Don't match on `any` here - since it's a simple type (String) we should be able to capture the specific argument we're looking for.